sklearn/feature_extraction/text.py

```diff
@@ -1474,13 +1479,25 @@ class TfidfVectorizer(CountVectorizer):
 
     See also
     --------
-    CountVectorizer
-        Tokenize the documents and count the occurrences of token and return
-        them as a sparse matrix
-
-    TfidfTransformer
-        Apply Term Frequency Inverse Document Frequency normalization to a
-        sparse matrix of occurrence counts.
+    CountVectorizer : Produces a sparse representation of the counts using
+        scipy.sparse.csr_matrix.
+
+    TfidfTransformer : Converts the count matrix from CountVectorizer to a
```

Comment:
In the context of TfidfVectorizer this is far too much detail.

Just say "Performs the TF-IDF transformation from a provided matrix of counts"